File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen = {}
        ans = []
        
        for i in range(len(nums)):
            print(nums[i])
            if (target - nums[i]) in seen.keys():
                print(seen)
                ans.append(seen[target-nums[i]])
                ans.append(i)
                break
            
            seen[nums[i]] = i
        
        return ans
        

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const seen = {};
    const ans = [];
    
    nums.forEach((num, index) => {
        if ((target - num) in seen) {
            ans.push(index, seen[target-num])
        } else {
            seen[num] = index;
        }
    });
    
    return ans;
};